.chloggen/tls.yaml

```diff
@@ -0,0 +1,9 @@
+change_type: "bug_fix"
```

Comment:
I think this is more of an enhancement, as there was not previously a non-functional TLS config.